xlml/utils/xpk.py

```diff
@@ -52,6 +53,12 @@ def _run_workload(
   """Run workload through xpk tool."""
   import subprocess
 
+  # Set the GCS path environment variable in the run_cmds
+  # TODO(jonbolin): Can't access `metric_config` from within
+  # task.(docker|kubernetes), so hardcode metric_config.SshEnvVars.GCS_OUTPUT
+  # here
+  run_cmds = f"export GCS_OUTPUT={gcs_path}; {run_cmds}"
```

Comment:
cc @RissyRan @will-cromar in case there's a better approach. `gcs_path` will be an XComArg, so we can't format outside of the `@task.docker`, and `metric_config` is inaccessible from within `@task.docker`.

Reply:
I think the solution here is to move this to a BashOperator and make XPK a dependency for the Composer env: https://cloud.google.com/composer/docs/composer-2/write-dags#bash-operator

I'll take this in a separate PR.

Reply:
Ah that makes sense, I guess xpk is still under rapid development so PyPi can fall behind. I'll stick with `git clone` for now. Will address in this PR: https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/208

Reply:
@RissyRan FYI I'm going to move the GCS support into https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/208 and rebase this PR once that's landed.